### PR TITLE
Keras/2.0.4 for Python 2.7.13

### PR DESCRIPTION
**Pull request by wpoely86**
Already present for Python 3.x.

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic66 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d0178b6d68d31cf81d45dceeefd4c47c for a full test report.

File: easybuild/easyconfigs/k/Keras/Keras-2.0.4-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Theano', '0.9.0', versionsuffix),
+    ('h5py', '2.7.0', '%(versionsuffix)s'),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+# it defaults to Tensorflow
+modextravars = {'KERAS_BACKEND': 'theano'}
```
Comment (boegel) [inline easybuild/easyconfigs/k/Keras/Keras-2.0.4-intel-2017a-Python-2.7.13.eb]: @wpoely86 please also do this in the existing easyconfig for Keras (just for the v2.0.4 one is fine imho)?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2507.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/01fa46b9b7e41c944e856599ed6002b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/52c0dc6837df4c0ec2b15c2844e99c4b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**